src/lib/UtapiClient.js

```diff
@@ -424,6 +437,9 @@ export default class UtapiClient {
                 ['incr', generateKey(p, action, timestamp)]
             );
         });
+        // We track the number of commands needed for each `paramsArr` object to
+        // eventually locate each group in the results from ioredis.
+        const commandsGroupSize = 2;
```

Comment:
It should be the length of the group of commands pushed to `cmds` for each object in `paramsArr`. There is a problem when locating the values in the response from Redis if `paramsArr.length` is > 2, at least in calculate their location using the `every` method here: https://github.com/scality/utapi/pull/117/files/26f297d3d7f986f1c1eec96da387bcd8551abee7#diff-ca6d1c871f1efa6d1be088e4ec103337L444